tests/testthat/test-compare-value.R

```diff
@@ -123,12 +123,18 @@ test_that("uses format method if available", {
     compare(factor(c("a", "b")), factor(c("a", "b"), levels = c("b", "a")))
   })
 })
-
 test_that("ignore_attr never uses format method", {
   expect_snapshot({
     compare(.POSIXct(1, "UTC"), .POSIXct(2, "UTC"), ignore_attr = TRUE)
   })
 })
+test_that("don't use format if numeric & within tolerance", {
+  dt <- as.POSIXct("2016-07-18 16:06:00", tz = "UTC")
+  expect_snapshot({
+    compare(dt, dt + 5)
```

Comment:
that means timestamps 5 seconds apart are considered equal? seems strange to me

Reply:
`compare()` never errors; see the snapshot for the actual results.

Reply:
in what sense are dt and dt+5 within 1e-8 of each other?

Reply:
The tolerance is relative, not absolute.

Reply:
(This behaviour isn't new; it's just exposed by the change to use `format`. I'll tackle it in a separate issue.)